### PR TITLE
New Category | Apps

### PR DESCRIPTION
**Pull request by Smutchings**
Added "Apps" as a category. May be useful to split into iOS and Android apps, though.

### CODE REVIEW

File: src/item/duolingo.md [hunk 1]
```diff
@@ -2,6 +2,6 @@
 title: 'Duolingo'
 link: 'https://www.duolingo.com'
 summary: 'Learn languages for free. Web or app.'
-tags: ['education', 'age-all']
+tags: ['education', 'apps' 'age-all']
```
Comment (hankchizljaw) [inline src/item/duolingo.md]: You're missing a comma here, so the build is failing ;)

**Comment by hankchizljaw** [comment]
Good shout!

File: src/item/duolingo.md (hunk 1 above)
Comment (Smutchings) [reply src/item/duolingo.md]: Added the comma, so this should now be working.

**Comment by hankchizljaw** [comment]
Lovely, thanks!

**Pull request merged**